src/sentry/event_manager.py

```diff
@@ -881,6 +881,13 @@ def _save_aggregate(self, event, hashes, release, **kwargs):
                 "group.created", skip_internal=True, tags={"platform": event.platform or "unknown"}
             )
 
+            group_created.send_robust(
```

Comment:
i would be seriously concerned about performance here since this is in pipeline

Reply:
where would be a better place to capture that a new group has been created?

Reply:
well its more than this his a high volume data concern, and its unclear we actually should be tracking an individual analytics event for every single issue (vs extrapolating in some other way)

similar concern if we tried to do this per-event

Reply:
There are orders of magnitude more events than there are issues. So it's not really the same degree of high volume. 

Is it not enough to just have the addition to signals at the project level-? Do we really need it in the event_manager, and would that mitigate the performance concern?

Reply:
Yes there are, but I dont love that we're duplicating data all over the place. It will add significantly to our bottom line over time and it will make it tough to justify the spend. This is something that could be batched out of kafka and/or pulled nightly from ETL, so why do we need it in raw event form?

Generally speaking, nothing should operate on an N scale w/ events (and issue are coupled to events). They are much more granular than we ever need from a bucketing point of view, and we optimize our prod infrastructure to manage that cost (without factoring in duplication of that data into our systems). If we begin duplicating this data or adding margins to that cost it will mess up our bottom line with our customers, which ultimately would mean we need to charge more (which we dont intend to do for the long tail).

Reply:
>Is it not enough to just have the addition to signals at the project level-? and would that mitigate the performance concern?

1) ^what about this part?

2) Also, if this is duplicated, tell me where to find it? I'm still trying to get the full picture of what we have, where it is, and whether it can be meaningfully joined to other things as needed for some of the questions we're trying to answer. 

AFAICT we have some gaps, and sometimes adding a field here or there would close them up. Sometimes deriving an answer is harder than it looks, and potentially more error-prone e.g. for issues because they're changing over time (splitting, merging, etc).

Reply:
> Is it not enough to just have the addition to signals at the project level-? and would that mitigate the performance concern?

I dont understand what you're suggesting here.

> Also, if this is duplicated, tell me where to find it? I'm still trying to get the full picture of what we have, where it is, and whether it can be meaningfully joined to other things as needed for some of the questions we're trying to answer.

```
SELECT COUNT(*) FROM sentry_groupedmessage WHERE [conditions]
```

Reply:
Thanks. 

In that case @smelancon @sydjryan is `sentry_groupedmessage` already on the list of things being added to bigquery-? @smelancon will that work for you if it gets added there-?

Reply:
I’m saying we should ETL it. We cannot put issues into big query because it’s 1) customer data, and 2) high Write volume

Reply:
we don't need the whole issue. I'm not sure what you mean re: ETL it? 

FWIW we only need a subset of fields, like counts of unique issues, the first_seen dates, resolved_at dates, and maybe one or two other things. We don't need any customer-specific data.

Reply:
I really just need a few columns from `sentry_groupedmessage` in bigquery to answer my questions. We could even aggregate somehow if that would help with the volume problem.